quic-newrelic-repolint.json

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
Probably should have renamed these files now the newrelic changes are upstream. I think it's confusing to have their company name in here :)

Reply:
> Probably should have renamed these files now the newrelic changes are upstream. I think it's confusing to have their company name in here :)

Yeah I noticed that, just wanted to get this merged, I need to update the config anyway.